### PR TITLE
Feature/winter drop

### PR DESCRIPTION
**Pull request by inotflying**
# [Winter Drop](https://minecraft.wiki/w/Winter_Drop)

- [ ] [Creaking Heart](https://minecraft.wiki/w/Creaking_Heart)
  - [ ] Implementation
  - [ ] Documentation
 - [ ] Pale Oak wood type
   - [x] Implementation
   - [ ] Documentation
- [ ] New foliage and vegetation blocks
  - [ ] Implementation
  - [ ] Documentation
- [ ] Resin
  - [x] Implementation
  - [ ] Documentation
- [x] Pale Garden biome

### CODE REVIEW

**Comment by celishere** [comment]
найди работу

**Comment by TwistedAsylumMC** [comment]
I think it's fine to not implement creaking blocks since we do not have entity support. If you get the chance to do the other foliage/vegation then that would be appreciated!

File: server/block/sign.go [hunk 1]
```diff
@@ -178,7 +177,10 @@ func (s Sign) NeighbourUpdateTick(pos, _ cube.Pos, w *world.World) {
 
 // EncodeBlock ...
 func (s Sign) EncodeBlock() (name string, properties map[string]any) {
-	woodType := strings.Replace(s.Wood.String(), "_", "", 1) + "_"
+	woodType := s.Wood.String() + "_"
+	if woodType == "dark_oak_" {
+		woodType = "darkoak_"
+	}
```
Comment (TwistedAsylumMC) [inline server/block/sign.go:183]: Can you compare s.Wood to DarkOakWood() instead, i'd also change the statement below to compare to OakWood()

**Review by TwistedAsylumMC — CHANGES_REQUESTED** [review]

File: server/item/resin_clump.go
Comment (DaPigGuy) [inline server/item/resin_clump.go:1]: Resin clump is also a block

**Comment by DaPigGuy** [comment]
I've cherry-picked 566d32bd3c88599d7080cd28ce9b8de9ba95c26c and 26c6f487c842f0f13de95b6721f0ee1a58e7e58a

**Comment by DaPigGuy** [comment]
I have cherry-picked and fixed-up the remaining two commits for Resin blocks/items. The Resin Clump item has been dropped for now as it needs additional work for the block.

Thanks for the contributions!

**Pull request closed without merging**